college_preprocessor.py

```diff
@@ -6,81 +6,180 @@
 import pika
```

Comment:
A descriptive comment at the top of the module explaining the general purpose of the module and it's process would be useful here, [here's an example from the survey preprocessor](https://github.com/rpiml/survey-preprocessor/blob/master/src/parse.js)